requirements/edx/custom.txt

```diff
@@ -9,12 +9,12 @@
 # This is required for A2E courses that were created with the temporary (xblock-drag-and-drop-v2-new) DnDv2 branch to continue to work.
 # FIXME: bump version to 2.1.6 when https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/154 merged and tagged
 -e git+https://github.com/open-craft/xblock-drag-and-drop-v2.git@82c9dc5e16d10793e8b79e60661e1a78893fce25#egg=xblock-drag-and-drop-v2-new
--e git+https://github.com/edx-solutions/xblock-ooyala.git@v2.1.4#egg=xblock-ooyala==2.1.4
+-e git+https://github.com/edx-solutions/xblock-ooyala.git@v2.1.5#egg=xblock-ooyala==2.1.5
 git+https://github.com/edx-solutions/xblock-group-project.git@0.1.2#egg=xblock-group-project==0.1.2
 -e git+https://github.com/edx-solutions/xblock-adventure.git@v0.1.1#egg=xblock-adventure==0.1.1
 -e git+https://github.com/open-craft/xblock-poll.git@v1.8.6#egg=xblock-poll==1.8.6
 -e git+https://github.com/open-craft/edx-notifications.git@9930a7069dd496e3229145d0dee8f750c9d1d7cc#egg=edx-notifications==0.8.4
```

Comment:
@moeez96 What is the reason to use https://github.com/open-craft/edx-notifications.git instead https://github.com/edx/edx-notifications.git?

Reply:
Im not sure about the reason or context. Looping in others. 
@msaqib52  @xitij2000 Can you please address this?

Reply:
It's probably to include a change from a PR that hasn't been merged upstream yet, but was needed here more urgently. 

It is also why a forked version of drag-and-drop-v2 is used here instead of the official released/tagged version.

Reply:
@xitij2000 Any idea when this change is expected to merge upstream? It would be great to merge that PR upstream.

Reply:
@ihtram this was required for the data deletion effort. While the changes are still waiting to be merged on the [upstream repository](https://github.com/edx/edx-notifications/pull/190), here we need to use an older version of edx-notification as the latest version isn't backwards compatible.